stroom-app/src/main/java/stroom/startup/SpringContexts.java

```diff
@@ -31,7 +32,7 @@ public SpringContexts() throws ClassNotFoundException {
         applicationContext = new AnnotationConfigWebApplicationContext();
         applicationContext.setParent(rootContext);
         applicationContext.registerShutdownHook();
-
```

Comment:
Adding this allows us to use the lifecycle annotations on our Spring components. Like @PostConstruct